### ISSUE TITLE
Bad McMaster.NETCore.Plugins version reference

### ISSUE THREAD

**Issue by Werebunny** [danielpalme/ReportGenerator#406]
**Describe the bug**
ReportGenerator.Core has a package resolve issue with McMaster.NETCore.Plugins because 1.1.118 doesn't exist on NuGet:

Error: ReportGenerator.Core 4.8.4 depends on McMaster.NETCore.Plugins (>= 1.1.118) but McMaster.NETCore.Plugins 1.1.118 was not found. An approximate best match of McMaster.NETCore.Plugins 1.2.0 was resolved.

This is otherwise known as NU1603, except VS Code with Omnisharp on Linux pops it up as an error (can be ignored).

**To Reproduce**
1. Add to project in VS Code:
```
<PackageReference Include="ReportGenerator.Core" Version="4.8.4" />
```

**Comment by danielpalme** [comment]
Will have a look within the next days.

**Comment by danielpalme** [comment]
Fixed in release 4.8.5
